MenuVuelo.cpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by alexteper on 20/7/20.
+//
+
+#include "MenuVuelo.h"
+
```

Comment:
Agregar include del .h
```suggestion
#include <iostream>
```